make/config/istio/istio-config-1.16.2.yaml

```diff
@@ -1,57 +0,0 @@
-apiVersion: install.istio.io/v1alpha1
-kind: IstioOperator
-metadata:
-  namespace: istio-system
-spec:
-  profile: "demo"
-  hub: gcr.io/istio-release
-  meshConfig:
-    # Change the following line to configure the trust domain of the Istio cluster.
-    trustDomain: foo.bar
-  values:
-    global:
-      # Change certificate provider to cert-manager istio agent for istio agent
-      caAddress: cert-manager-istio-csr.cert-manager.svc:443
-  components:
-    pilot:
-      k8s:
-        env:
-          # Disable istiod CA Sever functionality
-        - name: ENABLE_CA_SERVER
-          value: "false"
-        overlays:
```

Comment:
@MattiasGees Do we no longer need these overlays?

Reply:
I just found this: https://github.com/cert-manager/website/pull/1424

Reply:
yeah I had fixed it in Istio 1.16 but never found the time to update it until @MattiasGees reminded me!